website/docs/d/route53_delegation_set.html.markdown

```diff
@@ -0,0 +1,32 @@
+---
+layout: "aws"
+page_title: "AWS: aws_route53_delegation_set"
+sidebar_current: "docs-aws-datasource-route53-delegation-set"
```

Comment:
This documentation page will need a sidebar link in `website/aws.erb` 😄

Reply:
Added in my latest commit